docs/roadmap.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Scully roadmap
+published: true
+lang: en
+position: 5
+---
+
+# Scully roadmap.
+
+In here we are listing the things we would like to implement in Scully.
+We are giving no time-estimates as we are deppening on available time of contributors and/or sponsors.
+
+- release V2 of Scully
+  - Deprecate support for Angular before version 12.
+  - V2 will require Angular v12 and up
+  - V2 will switch over to the new Ivy library structure soon
+  - V1 will remain available for Angular versions 9/10/11
+  - support for versions before V9 is completely dropped.
+- Better control over server-start-stop.
+  - We have had some reports of stalling in CI/CD, an this seems to be the root cause of this issue.
+- Renderer build on top of [platform-server](https://angular.io/api/platform-server)
+  - Scully is now using Puppeteer to render the resulting pages. This approach will be using the platform server. While this puts some additional demands on the application (similar to Angular Universal, which uses the same engine), it can render pages _much_ faster. (It doesn't have to start a browser, navigate to a page and so on)
```

Comment:
Scully will still default to render with Puppeteer. Turning on Platform-server is opt-in.